src/main/java/org/springframework/integration/dsl/jms/Jms.java

```diff
@@ -128,4 +130,32 @@ public static JmsOutboundGatewaySpec outboundGateway(ConnectionFactory connectio
 		}
 	}
 
+	public static JmsListenerContainerSpec<DefaultMessageListenerContainer> container(
```

Comment:
Interesting how have we missed JavaDocs here...
Would you mind ti take care about them?
I just was going to say `@since 1.1` for these new methods, but see that no one of them has JavaDocs... :cry: